lms/djangoapps/grades/admin/admin.py

```diff
@@ -0,0 +1,24 @@
+"""
+Django admin page for grades models
+"""
+from django.contrib import admin
+
+from config_models.admin import ConfigurationModelAdmin, KeyedConfigurationModelAdmin
+
+from lms.djangoapps.grades.admin.models import CoursePersistentGradesFlag, PersistentGradesEnabledFlag
+from lms.djangoapps.grades.admin.forms import CoursePersistentGradesAdminForm
+
+
+class CourseAuthorizationAdmin(KeyedConfigurationModelAdmin):
+    """Admin for enabling subsection grades on a course-by-course basis."""
```

Comment:
I may be wrong, but I believe the edX guidelines is to always have multi-line docstrings.